components/TodoItem.tsx

```diff
@@ -0,0 +1,26 @@
+import { StyleSheet,Text,  } from "react-native";
```

Comment:
Please be sure to run prettier befire saving. As you see here there is trailing space and comma.